src/core/Form/Toggle/Toggle.baseStyles.tsx

```diff
@@ -8,86 +8,113 @@ const svgPrefix = 'icon-toggle_svg__';
 const iconWidth = '40px';
 const iconHeight = '24px';
 
+/* stylelint-disable no-descending-specificity */
 export const baseStyles = withSuomifiTheme(
   ({ theme }: TokensAndTheme) => css`
-  ${element({ theme })}
-  ${font({ theme })('bodyText')}
-  background-color: ${theme.colors.whiteBase};
+    ${element({ theme })}
+    ${font({ theme })('bodyText')}
+    background-color: ${theme.colors.whiteBase};
+    padding-left: 50px;
+    position: relative;
+    display: inline-block;
+
+    & .fi-toggle_icon-container {
+      position: absolute;
+      margin-right: ${theme.spacing.insetL};
+      left: 0;
+      top: 0.1em;
+    }
+
     & + .fi-toggle--with-button {
       &:focus {
         outline: 0;
-        ${focus({ theme, noPseudo: true })}
+        & .fi-toggle_icon-container {
+          ${focus({ theme, noPseudo: true })}
+          position: absolute;
+          &:after {
+            border-radius: 14px;
+            right: -4px;
+            left: -4px;
+          }
+        }
       }
       &:focus:not(:focus-visible) {
         outline: 0;
-        &:after {
-          content: none;
+        & .fi-toggle_icon-container {
+          &:after {
+            content: none;
+          }
         }
       }
     }
+
     & + .fi-toggle--with-input {
       &:focus-within {
         outline: 0;
-        ${focus({ theme, noPseudo: true })}
+        & .fi-toggle_icon-container {
+          ${focus({ theme, noPseudo: true })}
+          position: absolute;
+          &:after {
+            border-radius: 14px;
+            right: -4px;
+            left: -4px;
+          }
```

Comment:
Would it be cleaner to store this bit in a variable such as toggleFocusStyles and use that instead of repeating it for both variants? I think a case can be made for both approaches since it's only used here.

Reply:
This should now be fixed. Left the focus call out of the variable since it's easier to read that way in my opinion and the typings are a mess - including it would require a long custom type for some reason.